homeassistant/components/aladdin_connect/cover.py

```diff
@@ -28,25 +26,33 @@ async def async_setup_entry(
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up the Aladdin Connect platform."""
-    acc: AladdinConnectClient = hass.data[DOMAIN][config_entry.entry_id]
+    session: api.AsyncConfigEntryAuth = config_entry.runtime_data
+    acc = AladdinConnectClient(session)
     doors = await acc.get_doors()
```

Comment:
We can move the door fetching to `__init__.py` to prevent it fetching the doors twice.